packages/solutions/app-tools/src/index.ts

```diff
@@ -154,6 +154,19 @@ export default (
     i18n.changeLanguage({ locale });
 
     return {
+      async afterInit() {
```

Comment:
How about running the following codes in the body of `setup` function:

![image](https://github.com/web-infra-dev/modern.js/assets/7237365/626d11bb-9e7b-4052-a98c-b6c68c13795e)

Reply:
Oh It doesn't seem to work, the AppContext has not been initialized when the setup is executed,

Reply:
How about renaming `afterInit` to `beforeConfig`, because there is no `init` hook

Reply:
Okey, that's better~

Reply:
The `beforeConfig` should also be documented 😄

Reply:
done~